packages/shared-state-pirania/files/etc/shared-state/publishers/shared-state-publish_vouchers

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/lua
+
+--! LibreMesh
+--! Copyright (C) 2019  Marcos Gutierrez <gmarcos@altermundi.net>
+--! Copyright (C) 2019  Luandro <luandro@gmail.com>
+--!
+--! This program is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU Affero General Public License as
+--! published by the Free Software Foundation, either version 3 of the
+--! License, or (at your option) any later version.
+--!
+--! This program is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU Affero General Public License for more details.
+--!
+--! You should have received a copy of the GNU Affero General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+local fs = require("nixio.fs")
+local JSON = require("luci.jsonc")
+
+
+local function shell(command)
+    -- TODO(nicoechaniz): sanitize or evaluate if this is a security risk
```

Comment:
This comment seems to not apply here, did you copied this function from another place?

Reply:
I use this function in many places, it was a function that Nico originally wrote and that for future references and replacement I copy as is. I remove the comment anyway.